splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/joins/BroadcastJoinIT.java

```diff
@@ -631,9 +631,9 @@ public void testUncomparableCharColumnsBroadCastJoin() throws Exception {
                 " on s1.b1 = s2.a2" , useSpark
         ) ;
         try (ResultSet rs = classWatcher.executeQuery(sqlText)) {
-            Assert.fail("Exception not thrown");
+
         } catch (SQLSyntaxErrorException e) {
```

Comment:
You can remove the catch if a raised exception is cause for failure

Reply:
Removed the catch.